range_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from range import bucket_amperes, bucket_individual_readings, report_consecutive_groups, check_error, input_to_amps
+
+class TestAmperesRanges(unittest.TestCase):
+    def test_two_values_are_reported_in_range(self):
```

Comment:
Test cases need to be enhanced with variations in test set values.
empty set, single value set, elements with repetition needs to be considered.